src/shared/lib/StoreUtils.ts

```diff
@@ -465,6 +465,30 @@ export async function fetchCnaOncoKbData(uniqueSampleKeyToTumorType:{[uniqueSamp
     }
 }
 
+export async function fetchCnaOncoKbDataWithGeneMolecularData(uniqueSampleKeyToTumorType:{[uniqueSampleKey: string]: string},
+                                         annotatedGenes:{[entrezGeneId:number]:boolean},
+                                         geneMolecularData:MobxPromise<GeneMolecularData[]>,
+                                          molecularProfileIdToMolecularProfile:{[molecularProfileId:string]:MolecularProfile},
+                                         client: OncoKbAPI = oncokbClient)
+{
+    if (!geneMolecularData.result || geneMolecularData.result.length === 0) {
+        return ONCOKB_DEFAULT;
+    }
+    else
+    {
+        const alterationsToQuery = _.filter(geneMolecularData.result, d=>{
```

Comment:
i think d is MolecularData. we shoudl just name is molecularData or some such.  so much easier to read

Reply:
sure